web/docs/migrate-from-0-13-to-0-14.md

```diff
@@ -0,0 +1,260 @@
+
+:::
+
+## What's new in 0.14.0?
+
+### Better auth user API
+
+We introduced a much simpler API for accessing user auth fields like `username`, `email` or `isEmailVerified` on the `user` object. You don't need to helper functions every time you want to access the user's `username` or extra steps to get proper typing.
+
+## How to migrate?
+
+### Migrate how you access user auth fields
+
+We had to make a couple of breaking changes to reach the new simpler API. 
+
+Follow the steps below to migrate:
+
+1. **Replace `getUsername` use** with `user.identities.username.id`
+
+    If you didn't use `getUsername` in your code, you can skip this step.
+
+    <Tabs>
+    <TabItem value="before" label="Before">
+
+    ```tsx title="src/MainPage.tsx"
+    import { getUsername, AuthUser } from 'wasp/auth'
+
+    const MainPage = ({ user }: { user: AuthUser }) => {
+        const username = getUsername(user)
+        // ...
+    }
+    ```
+
+    ```ts title=src/tasks.ts
+    import { getUsername } from 'wasp/auth'
+
+    export const createTask: CreateTask<...>  = async (args, context) => {
+        const username = getUsername(context.user)
+        // ...
+    }
+    ```
+
+    </TabItem>
+    <TabItem value="after" label="After">
+
+    ```tsx title="src/MainPage.tsx"
+    import { AuthUser } from 'wasp/auth'
+    const MainPage = ({ user }: { user: AuthUser }) => {
+        const username = user.identities.username?.id
+        // ...
+    }
+    ```
+
+    ```ts title=src/tasks.ts
+    export const createTask: CreateTask<...>  = async (args, context) => {
+        const username = context.user.identities.username?.id
+        // ...
+    }
+    ```
+    </TabItem>
+    </Tabs>
+
+1. **Replace `getEmail` use** with `user.identities.email.id`
+
+    If you didn't use `getEmail` in your code, you can skip this step.
+
+    <Tabs>
+    <TabItem value="before" label="Before">
+
+    ```tsx title="src/MainPage.tsx"
+    import { getEmail, AuthUser } from 'wasp/auth'
+
+    const MainPage = ({ user }: { user: AuthUser }) => {
+        const email = getEmail(user)
+        // ...
+    }
+    ```
+
+    ```ts title=src/tasks.ts
+    import { getEmail } from 'wasp/auth'
+
+    export const createTask: CreateTask<...>  = async (args, context) => {
+        const email = getEmail(context.user)
+        // ...
+    }
+    ```
+
+    </TabItem>
+    <TabItem value="after" label="After">
+
+       ```tsx title="src/MainPage.tsx"
+    import { AuthUser } from 'wasp/auth'
+
+    const MainPage = ({ user }: { user: AuthUser }) => {
+        const email = user.identities.email?.id
+        // ...
+    }
+    ```
+
+    ```ts title=src/tasks.ts
+    export const createTask: CreateTask<...>  = async (args, context) => {
+        const email = context.user.identities.email?.id
+        // ...
+    }
+    ```
+    </TabItem>
+    </Tabs>
+
+1. **Replace any other auth field usage** with `user.identities.<provider>.<value>`
```

Comment:
I dind't quite get this. What got deprecated? `findUserIdentity`? What else? Any way to phrase this in a bit simpler way? Maybe I am just missing context and would understand this if actually used this older code?

Reply:
I've updated the phrasing to be clear that it's related to `providerData`